src/libraries/System.Text.Json/src/System/Collections/Generic/StackExtensions.netstandard.cs

```diff
@@ -3,34 +3,35 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
 using System.Runtime.CompilerServices;
 
 namespace System.Collections.Generic
 {
     /// <summary>Polyfills for <see cref="Stack{T}"/>.</summary>
     internal static class StackExtensions
     {
-        public static bool TryPeek<T>(this Stack<T> stack, out T result)
+        public static bool TryPeek<T>(this Stack<T> stack, [MaybeNullWhen(false)] out T result)
         {
             if (stack.Count > 0)
             {
                 result = stack.Peek();
                 return true;
             }
 
-            result = default;
+            result = default!;
```

Comment:
Why do we need to bang this when we already annotated that T could be null?

Reply:
the compiler used to complaing for setting default for `[MaybeNull]` fields, but seems that bug is fixed.